content/en/tools/ledgergraph/ledgergraph-checksums.md

```diff
@@ -15,6 +15,8 @@ You can download the LedgerGraph release files if you have access to the reposit
 
 Before you install the downloaded files, verify the integrity of each file to ensure that it has not been tampered with or corrupted during the download. You can do this by calculating the **checksum** of your downloaded file, then comparing its value to the checksum value provided below.
 
+## Checksums of LedgerGraph 1.2.4 release files
```

Comment:
shouldn't we have a list here below the sub-title ?

Reply:
Yes, I will add the checksums this morning once I get them, I was just setting up the file.